license

```diff
@@ -0,0 +1,7 @@
+ISC License
```

Comment:
I thought so too, but previously it was "ISC" in package.json, so maybe there was a reason for it.